src/components/atoms/input/index.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from "prop-types";
+import useInput from "../../../hooks/useInput";
+import { createElement } from "react";
+
+const Input = (props) => {
+  const {
+    as = "input",
+    className,
+    type,
+    required,
+    name,
+    secret,
+    length,
+    maxLength,
+    min,
+    max,
+    digit,
+    placeholder,
+  } = props;
```

Comment:
기본으로 받는 prop들(검증이 딱히 필요없는)은 아래와 같은 방식으로 작성해도 좋을거 같다는 생각이 드네요

```suggestion
  const {
    // (생략)
    ...rest
  } = props;
```

Reply:
이렇게 prop을 받고 render할 때 아래처럼 render해준다면 `<input>` 태그가 기본으로 받을 수 있는 모든 prop을 받을 수 있어 보다 유연하게 사용가능할거 같네요

```jsx
<input 
  // ... (생략)
  {...rest}
/>
```

Reply:
아! 
저도 만들면서 props가 너무 길어지는 경우가 간혹 발생해서 고민이었는데 이런 방법이!